examples/dtt/CMakeLists.txt

```diff
@@ -1,21 +1,12 @@
 file(GLOB files_dtt_x fftw_f03_dtt_x.f90)
-file(GLOB files_dtt_z fftw_f03_dtt_z.f90)
-
-include_directories(${CMAKE_SOURCE_DIR}/src)
-include_directories(${FFTW_INCLUDE_DIRS})
+set(case "dtt_x")
+set(case_dir "${test_dir}/${case}")
+CreateMPITest("${case_dir}" "${case}" "${files_dtt_x}" "" "" "" "1")
 
-add_executable(dtt_x ${files_dtt_x})
-add_executable(dtt_z ${files_dtt_z})
+file(GLOB files_dtt_z fftw_f03_dtt_z.f90)
+set(case "dtt_z")
+set(case_dir "${test_dir}/${case}")
+CreateMPITest("${case_dir}" "${case}" "${files_dtt_z}" "" "" "" "1")
 
-target_link_libraries(dtt_x PRIVATE decomp2d examples_utils)
-target_link_libraries(dtt_z PRIVATE decomp2d examples_utils)
+#include_directories(${FFTW_INCLUDE_DIRS})
```

Comment:
Can we delete this line or is it required for testing with FFTW?

Reply:
Yes I just kept since I was adding the line in the macro. Just forgot to remove it.